modules/widgets/src/stylesheet.css

```diff
@@ -31,3 +31,24 @@
   pointer-events: auto;
   cursor: pointer;
 }
+
+.deckgl-widget-button {
+  color: white;
```

Comment:
The icon color is meant to them-able. Let's do `var(--icon-color, white)` and define the appropriate color in `themes.ts`. I'll do this in the fullscreen PR for you to pull in.

Reply:
I'll also add `--icon-compass-north-color` and `--icon-compass-south-color`